src/app/feature/calendar/core/snowflakes/snowflakes.component.ts

```diff
@@ -32,18 +31,16 @@ export class SnowflakesComponent {
   }
 
   private letItSnow(): void {
-    this.zone.runOutsideAngular(() => {
```

Comment:
Still a bit unclear for me if this is needed to be removed as currently zoneless is kind of "hybrid" right now.
Angular Material is zoneless compatible but still contains a lot of `runOutsideAngular`.

Maybe I should add it back?

Any input on this @JeanMeche?

Reply:
This app still relies on Zone.js, but it doesn't a call on a patch API. This should be OK.

Reply:
Well I removed zone.js in [this commit](https://github.com/danielglejzner/angular-christmas-calendar/pull/86/commits/0dcb4856db3b473ed6c86438b1ee94382d8e9070) ^^

Reply:
Oh perfect!
 
Also libraries will still rely on `runOutsideAngular` when they want to support both zoneless & zonefull apps. 
For app, `runOutsideAngular` is a noop when we use the zoneless provider.

Reply:
Ohhhhhhh okay, I get it I think, thanks!